### ISSUE TITLE
SmartCity-Accessibility Server Side #440 failed

### ISSUE THREAD

**Issue by JuanJenkinsBot** [TechnionYP5777/SmartCity-Accessibility#283]
Build 'SmartCity-Accessibility Server Side' is failing!

Failed Tests :
```
No tests ran.
```

Changes since last successful build:
- [yaelAmitay] 93bc166dbeb22d143a0d39070b496ebfac36ebc0 - signup server side #253

[View full output](http://kapalex.ddns.net:8080/job/SmartCity-Accessibility%20Server%20Side/440/)

**Comment by JuanJenkinsBot** [comment]
Build was fixed!
